integration_tests/mockApis/frontendComponentApi.ts

```diff
@@ -30,7 +77,7 @@ const stubGetFooterComponent500error = (): SuperAgentRequest =>
     },
   })
 
-const stubGetHeaderComponent = (): SuperAgentRequest =>
+const stubGetHeaderComponent = (name = 'J. Smith'): SuperAgentRequest =>
```

Comment:
I've given the stub a default name parameter that will be used in the stubbed markup on line 123 below 👇 
And then tests can override it if they need to (as we are doing in the login.cy.ts test above ☝️ )